visualization.shadertoy/screensaver.shadertoy.txt

```diff
@@ -0,0 +1 @@
+visualization.shadertoy https://github.com/notspiff/visualization.shadertoy master
```

Comment:
visualization.shadertoy/visualization.shadertoy.txt 
because it is not a screensaver. is it ?